opentelemetry/src/logs/record.rs

```diff
@@ -373,10 +373,13 @@ impl LogRecordBuilder {
     }
 
     /// Sets the `event_name` of a record.
-    pub fn with_name(self, name: Cow<'static, str>) -> Self {
+    pub fn with_name<T>(self, name: T) -> Self
```

Comment:
this is unrelated?

Reply:
Oh I made the change while I wrote unit testing IIRC. This make it easier for users to input `&'static str`